src/hammer-vlsi/generate_properties.py

```diff
@@ -189,6 +200,7 @@ def main(args) -> int:
     generate_interface(HammerDRCTool)
     generate_interface(HammerLVSTool)
     generate_interface(HammerSRAMGeneratorTool)
+    generate_interface(HammerPCBDeliverableTool)
```

Comment:
Is there a reason not to just call it `HammerPCBTool`?

Reply:
Well, I used this to clarify that it's not building a PCB. I am personally fine with `HammerPCBTool`

Reply:
Yeah I think we want to be careful not to accidentally claim to build the PCB itself